internal/inventory/asset.go

```diff
@@ -161,7 +181,11 @@ type Event struct {
 }
 
 type Network struct {
-	Name string `json:"name,omitempty"`
+	Name      string `json:"name,omitempty"`
+	Direction string `json:"direction,omitempty"`
+	Type      string `json:"type,omitempty"`
+	// Protocol string `json:"protocol,omitempty"`
+	// Transport []string `json:"transport,omitempty"`
```

Comment:
Is this a leftover from development or future-proofing?

Reply:
we might be able to add those but i'll remove it from this PR. maybe next iteration